src/styles/css/index.css

```diff
@@ -1,2 +1,870 @@
-h1 {
-  color: #0fe05f; }
+@import url("https://fonts.googleapis.com/css?family=Open+Sans");
+html,
+body,
```

Comment:
@DennisWanjiru I still think we don't need this file at the moment. Let's avoid going ahead of our selves and write code to be used for the moment.

Reply:
Yeah @j33n. I forgot to recompile the sass. Yeah, am removing it right away. Thanks for the highlight.